screen_brightness_control/__init__.py

```diff
@@ -17,6 +17,13 @@
 _logger = logging.getLogger(__name__)
 _logger.addHandler(logging.NullHandler())
 
+# Global variable to control whether duplicate monitors are allowed
+# Duplicate monitor info could occur due to two known reasons
+# 1. The same monitor is connected through different interfaces (e.g., HDMI, DisplayPort, DVI, VGA) at the same time.
+# 2. Different monitors share the exactly same identifiers, including EDID, serial number and name, due to careless manufacturer.
+# In case #1, duplicate monitor info is considered redundant and should be filtered out.
+# In case #2, it is essential and should be preserved.
+ALLOW_DUPLICATES = False
```

Comment:
Not sure if having this via kwargs and global variables is ideal, especially since changing this variable won't change what the kwarg defaults to.
```python
>>> import inspect
>>> inspect.signature(sbc.list_monitors_info)
<Signature (method: Optional[str] = None, allow_duplicates: bool = False, unsupported: bool = False) -> List[dict]>
>>> sbc.ALLOW_DUPLICATES = True
>>> inspect.signature(sbc.list_monitors_info)
<Signature (method: Optional[str] = None, allow_duplicates: bool = False, unsupported: bool = False) -> List[dict]>
```
I think this would get pretty confusing for users and that maybe instead an `allow_duplicates` kwarg should be added to relevant functions and propagated by them, although that is a pretty long list:
- `get_brightness`
- `set_brightness`
- `fade_brightness`
  - Will be absorbed into `**kwargs` automatically but needs to be passed on to `filter_monitors`
- `filter_monitors`
- `__brightness`
- `list_monitors`